### PR TITLE
Track clicks to Invoice Pay Now button

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Track clicks to Invoice Pay Now button

## Motivation and Context
Billing & Invoices epic

## How Has This Been Tested?
[stage-1] with authenticated and unauthenticated users.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: web/scripts/billing/services/svc-billing-factory.js [hunk 1]
```diff
@@ -127,6 +127,14 @@ angular.module('risevision.apps.billing.services')
       };
 
       factory.payInvoice = function () {
+        analyticsFactory.track('Invoice Pay Now Clicked', {
```
Comment (alex-deaconu) [inline web/scripts/billing/services/svc-billing-factory.js:130]: @ezequielc this is contrary to the pattern we have in other factories where we track the event after the action is completed (https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/purchase/services/svc-purchase-factory.js#L253). Also, if you concur with the above, could we move the code to the billingFactory with this occasion and add a factory function (https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/billing/services/svc-billing-factory.js#L111)? Thanks.

Comment (ezequielc) [reply web/scripts/billing/services/svc-billing-factory.js:130]: @alex-deaconu Thanks for the feedback, the name of that event threw me off (Order Pay Now Clicked). I agree it is a good idea to track it once it has completed, but I would suggest to rename the event to 'Invoice Paid' instead, what do you think?

I am not sure what do you mean by moving the code to the billingFactory, it is already in billingFactory. But yeah, I can create a trackInvoicePaid() function for it. Let me know if that is what you meant. Thanks!

Comment (alex-deaconu) [reply web/scripts/billing/services/svc-billing-factory.js:130]: @ezequielc oops, disregard re: moving to billingFactory, I thought it was in the controller. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Updated to trigger the event only after successful payment and renamed the event for clarity. Please review. Thanks

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good, thanks for updating 👍

**Pull request merged**